saltstack/base/salt/top.sls

```diff
@@ -16,6 +16,7 @@ base:
     - ccmv2
     - custom
     - mount
+    - luks
```

Comment:
Like for `packages.sls`, I wonder if this should be placed in a condition block checking for RHEL8.

But I guess this is not overly critical. `reopen-luks-volume.service` and the `pip` install of `aws-encryption-sdk-cli` are likely compatible with CentOS7 as well, even if never actually used later. 😄

Reply:
What about cloud providers? Isn't this AWS_GOV only?